src/components/TeamMemberCard/TeamMemberCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC, useEffect, useState } from 'react'
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent'
+import CardMedia from '@mui/material/CardMedia'
+import Typography from '@mui/material/Typography'
+import TeamMember from '@/types/TeamMember'
+import i18next from 'i18next'
+import OptionalLinkWrapper from '../OptionalLinkWrapper/OptionalLinkWrapper';
+
+interface TeamMemberCardProps {
+    member: TeamMember
+}
+
+const TeamMemberCard: FC<TeamMemberCardProps> = ({ member }) => {
+    const [name, setName] = useState(member.nameEN)
+    const [title, setTitle] = useState(member.titleEN)
+
+    useEffect(() => {
```

Comment:
🤔  There is still a reactivity issue where when switching the page, the names don't reset. I don't know how to solve it quite yet, but I think we can make a follow-up issue to iteratively improve

Reply:
I'm okay with addressing this in a separate PR

Reply:
Oh I was able to solve it this morning, forgot to update